ec2/spark_ec2.py

```diff
@@ -569,15 +569,34 @@ def launch_cluster(conn, opts, cluster_name):
         master_nodes = master_res.instances
         print "Launched master in %s, regid = %s" % (zone, master_res.id)
 
```

Comment:
Revisiting this now, I wonder if it would just be simpler all around to insert a 5-second sleep right here and call it day. That should give AWS enough time to get its act together.

Having the wait/retry logic all over the place seems more a burden than a benefit, honestly. What do you think?